easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -83,6 +95,10 @@ def configure_step(self, srcdir=None, builddir=None):
                 srcdir = default_srcdir
 
         options = ['-DCMAKE_INSTALL_PREFIX=%s' % self.installdir]
+
+        if '-DCMAKE_BUILD_TYPE' not in self.cfg['configopts']:
+            options.append("-DCMAKE_BUILD_TYPE=%s" % self.cfg['buildtype'])
```

Comment:
sure, but it may not be clear where the `-DCMAKE_BUILD_TYPE` suddenly comes from, or why `build_type` is not being picked up...